include/envoy/runtime/runtime.h

```diff
@@ -220,6 +220,17 @@ class Snapshot {
    */
   virtual ConstStringOptRef get(absl::string_view key) const PURE;
 
+  /**
+   * Fetch an integer runtime key. Runtime keys larger than ~2^53 may not be accurately converted
+   * into integers and will return default_value.
+   * @param key supplies the key to fetch.
+   * @param default_value supplies the default value that will be used if either the feature key
+   *        does not exist or it is not a fractional percent.
+   * @return uint64_t the runtime value or the default value.
+   */
+  virtual uint64_t getInteger(absl::string_view key,
```

Comment:
This is confusing. You are getting an integer but the default is a fractional percent? Can you change the name or clarify somehow?

Reply:
I need a name that captures the idea "Get me the integer value of a fractional percent". Would `getIntegerValueOfFractionalPercent` be too long-winded?

Reply:
What does an integer value of a fractional percent mean? Are you getting the numerator? I don't see how this would otherwise be anything other than 0 or 1. ;)

Reply:
Good point- I'll use `getNumeratorOfFractionalPercent`.